packages/messaging/README.md

```diff
@@ -24,36 +24,39 @@ cd MyReactApp
 npm install @adobe/react-native-aepmessaging
 ```
 
-**Installation instructions for `@adobe/react-native-aepmessaging` v1.0.0-beta.2 with in-app messaging support**  
-Download the node package `adobe-react-native-aepmessaging-1.0.0-beta.2.tgz` from [github release](https://github.com/adobe/aepsdk-react-native/releases/tag/%40adobe%2Freact-native-aepmessaging%401.0.0-beta.2) and save it in a folder.  
+**Installation instructions for `@adobe/react-native-aepmessaging` v1.0.0-beta.3 with in-app messaging support**   
 Install the `@adobe/react-native-aepmessaging` package:
 
 ```bash
 cd MyReactApp
-npm install {path to node package file}
+npm install @adobe/react-native-aepmessaging
```

Comment:
User should install from the instruction in prerelease md?
Do we want to user to use npm install @adobe/react-native-aepmessaging?

Reply:
There is a version available for this they can install, although if they want more than extension version, they would do a local install so I'm fine going either way